torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4829,6 +4835,15 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            supports_gradgrad=True,
            supports_out=False,
            autodiff_nonfusible_nodes=["aten::hardtanh"]),
+    OpInfo('nn.functional.gelu',
+           aten_name="gelu",
+           supports_autograd=True,
+           assert_autodiffed=True,
+           sample_inputs_func=sample_inputs_gelu,
+           dtypesIfCUDA=floating_types_and(torch.half, torch.bfloat16),
+           supports_gradgrad=True,
+           supports_out=False,
+           autodiff_nonfusible_nodes=["aten::gelu"]),
```

Comment:
this is wrong, we are fusing `aten::gelu`, maybe you need to rebase or something ?

Reply:
fusing means `FusionGroup` not DifferentiableGraph. We aren't fusing individual ops for these tests. We could but this would be a much bigger change.